packages/gatsby-transformer-remark/README.md

```diff
@@ -81,6 +86,22 @@ Using the following GraphQL query you'll be able to get the table of contents
 
 By default the tableOfContents is using the field `slug` to generate URLs. You can however provide another field using the pathToSlugField parameter. **Note** that providing a non existing field will cause the result to be null.
 
+To configure tableOfContents by graphQL query, adjust query as shown above. To pass default options to the plugin generating the tableOfContents, configure it as shown below. The options shown below are the defaults used by the plugin.
+
+```javascript
+// In your gatsby-config.js
+plugins: [
+  {
+    resolve: `gatsby-transformer-remark`,
+    options: {
+      heading: null,
+      maxDepth: 6,
+      tight: null,
```

Comment:
can we group those under `toc` or `tableOfContents` object?

Reply:
fixed in new commit

Reply:
I mean in options that user sets:
```
plugins: [
  {
    resolve: `gatsby-transformer-remark`,
    options: {
	  tableOfContents: {
        heading: null,
        maxDepth: 6,
        tight: null,
      }
     },
  },
]
```